nilearn/plotting/find_cuts.py

```diff
@@ -33,7 +33,7 @@ def find_xyz_cut_coords(img, mask=None, activation_threshold=None):
         -----------
         img : 3D Nifti1Image
             The brain map.
-        mask : 3D ndarray, boolean, optional
+        mask : 3D ndarray, boolean or 3D Niimg-like object, optional
```

Comment:
This parameter should take only Nifti images and should be renamed from mask -> mask_img.

```mask_img : 3D Nifti1Image, optional```

Should be added a bit of documentation as,  "An optional brain mask, provided mask_img should not be empty."